lib/screens/signup_screen.dart

```diff
@@ -209,6 +241,14 @@ class _SignupScreenState extends State<SignupScreen> {
                     _confirmP = true;
                 });
                 log('$_email');
+                res = await ref
+                    .read(signupRepositoryProvider)
+                    .signup(_userNametext.text, _passwordtext.text);
+
+                if (res.statusCode == 200) {
+                  Navigator.of(context).pushNamed('loginPage');
+                  print("Yes!!");
+                }
```

Comment:
Same as above. Move logic out of this file and use something such as FutureProvider